accepted/future-releases/named-arguments-anywhere/feature-specification.md

```diff
@@ -80,13 +80,13 @@ To update the current specification, the section:
 
 becomes:
 
-> Let *L* be an arguments list of the form <code>(*p*<sub>1</sub>, , *p*<sub>*k*</sub>)</code> with *m* positional arguments, *p*<sub>*q*<sub>1</sub></sub>  *p*<sub>*q*<sub>m</sub></sub> (in source order) and *n* named arguments, *p*<sub>*d*<sub>1</sub></sub>  *p*<sub>*d*<sub>n</sub></sub> (also in source order), so that *k = m + n*.
```

Comment:
This is actually weird. My original had ellipsis..es (plural is what?) there, the "…" character. Must have gotten lost in cut-n-pasting. Should *not* have happened.
(I refer to use the single "…" character instead of three "." characters in non-source-code text where possible).

I'll fix these.